lib/src/test/kotlin/com/swmansion/starknet/data/TransactionHashCalculatorTest.kt

```diff
@@ -12,7 +12,7 @@ internal class TransactionHashCalculatorTest {
     inner class DeprecatedTransactionHashTest {
         private val calldata = listOf(Felt(999), Felt(888), Felt(777))
         private val maxFee = Felt.fromHex("0xabcd987654210")
-        private val chainId = StarknetChainId.GOERLI
+        private val chainId = StarknetChainId.SEPOLIA
```

Comment:
Nit: I'm a bit unsure about this change. If I remember correctly, the previous hashes were verified against some tool like sn.js.

Did you verify the new ones? Maybe we should just use `StarknetChainId.fromHex("SN_GOERLI")` to avoid changing the test cases.

@THenry14 WDYT?

Reply:
I think we should be pretty safe, we can verify to be double-sure but I suppose this should be fine as is (as we do not change any logic behind calculations)